src/_data/capability-matrix.yml

```diff
@@ -202,24 +202,24 @@ categories:
             l1: 'Yes'
             l2: fully supported
             l3: ''
-      - name: Aggregators
+      - name: Metrics
         values:
           - class: model
             l1: 'Partially'
             l2: user-provided metrics
-            l3: Allow transforms to aggregate simple metrics across bundles in a <tt>DoFn</tt>. Semantically equivalent to using an additional output, but support partial results as the transform executes. Will likely want to augment <tt>Aggregators</tt> to be more useful for processing unbounded data by making them windowed.
+            l3: Allow transforms to gather simple metrics across bundles in a <tt>PTransform</tt>. Provide a mechanism to obtain both committed and attempted metrics. Semantically equivalent to using an additional output, but support partial results as the transform executes. Will likely want to augment <tt>Metrics</tt> to be more useful for processing unbounded data by making them windowed.
```

Comment:
Committed metrics are semantically equivalent to using an additional output, right? Not attempted.